documentation/GuideLines.md

```diff
@@ -0,0 +1,165 @@
+# DRAFT
+This is a draft of the proposed code standards and best practices for the Mission Control Project going forward to help facilitate a productive cross-team workflow to a unified standard. The plan is to use this standard to outline the workflow milestones for the more long term restructuring of some components needed to fulfill the vision of Mission Control envisioned by the stakeholder.
+
+# GitFlow - Git pull requests practices and standards
+### Create a branch
+* Pull down from Master to your local machine. This will assure that you have the latest and greatest from the repo!
+```
+git pull master
+```
+* Create/checkout a local feature branch. Once you have the master locally, create a branch of your feature.
+```
+git checkout -b feature/"YOUR-FEATURE-NAME"
```

Comment:
it would be good to give an example of a feature

Reply:
Based on this feed back have added an ```Example Feature``` sub section here.